### ISSUE TITLE
Add albums/playlists duration

### ISSUE THREAD

**Issue by KGriss** [vfsfitvnm/ViMusic#85]
### Feature description

A small quality-of-life add could be to have the album and playlist duration shown on the description screen, next to the publishing date. I don't know if this information is available on the YouTube Music API but it can be done by simply make the sum of the durations of songs in the list.

### Why do you want this feature?

It is always appreciated to know how long is the album you're about to start.

**Comment by Malopieds** [comment]
Isn't the data always in English due to the "en" value for String in the Client?

**Comment by vfsfitvnm** [comment]
Yes, but that won't work if I provide the ability to change the client language (thought I won't do so, but that was a decision I made at the beginning)
